config/betamocks/services_config.yml

```diff
@@ -854,3 +854,13 @@
         :uid_location: "url"
         :uid_locator: "\/services/claims/v2/veterans\/(?:.+)\/claims\/(.+)/5103"
 
+  # Lighthouse Benefits Documents API
+- :name: "Lighthouse Benefits Documents"
+  :base_uri: <%= "#{URI(Settings.lighthouse.benefits_documents.host).host}:#{URI(Settings.lighthouse.benefits_documents.host).port}" %>
+  :endpoints:
+    - :method: :post
+      :path: "/benefits-documents/v1/documents"
+      :file_path: "/lighthouse/benefits_documents"
+      :cache_multiple_responses:
+        :uid_location: "url"
+        :uid_locator: "\/benefits-documents/v1/documents"
```

Comment:
I'm not totally sure if I put the correct value here. I copied one of the `post` examples from above

Reply:
I typically see the `cache_multiple_responses` used for describing where the query/URL params are located. In this case, `POST /benefits-documents/v1/documents` doesn't have any query or URL parameters, so I _think_ we can leave off the `:cache_multiple_responses` block

Reply:
Updated